.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-1.9.3
```

Comment:
I think if we develop at lowest version supported is a good idea, this lock prevents the use of unavailable resources at 1.9.3